### ISSUE TITLE
I want the gboard layout

### ISSUE THREAD

**Issue by Joshuasony** [AnySoftKeyboard/AnySoftKeyboard#2566]
I would love to be able to use a gboard layout for AnySoftKeyboard. I know there are others who want the same. 

Anybody out there who can help?

My main issues are the following:
- space button is too small
- space button should be configurable to be aligned slightly to the right or left (not centered)
- as for the bottom row, I need special character, comma, emoji, space and period and submit (in that order)

The following options from the app are always just missing one of those things:
- iPhone styled (no mic): misses emoji button and space bar is positioned too far to the left
- Alternate (no mic): misses special character button
- Simple: misses emoji button and space bar is centered

![gboard](https://user-images.githubusercontent.com/7139459/99877035-9a460d80-2bfb-11eb-86d4-8109d591a997.png)

**Comment by Mageek627** [comment]
I agree. That would make the transition from Gboard easier and smoother.

**Comment by teynav** [comment]
I think present layout makes it more bloated , even the symbol space of the present keyboard contains so many symbols that are unusable for normal day to day usage Gboard minimalistic approach with/without symbols seems more intuitive than anything AnySoftKeyboard has to offer

**Comment by felschr** [comment]
I'm really missing this layout, and I'd like to offer 20 $ for an implementation.

**Comment by robsonsobral** [comment]
I want SwiftKey layout! It's awaful to find the keys after so many years of SwiftKey.

**Comment by felschr** [comment]
A Gboard layout  seems to have been added in
https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/2147

There just hasn't been a new ASK release on F-Droid for quite a while. And the Play Store didn't show any updates because I'm guessing it's using a different signature or similar.

**Comment by gabrc52** [comment]
Hmm, the latest release didn't install, it's a downgrade or was signed with different keys?
